sda-commons-cloudevents/build.gradle

```diff
@@ -0,0 +1,12 @@
+dependencies {
+
+  api ('org.springframework.boot:spring-boot-starter-validation')
```

Comment:
Do we really need this dependency for production? I would prefer to only include `javax.validation` which is referenced in imports.